Formula/aptos.rb

```diff
@@ -0,0 +1,43 @@
+class Aptos < Formula
```

Comment:
Can we be more specific to use `aptos-cli` instead of `aptos`?
```suggestion
class AptosCli < Formula
```

Reply:
I think we want to leave it at Aptos for now to be consistent with the crate and binary.

Reply:
Sounds good 👍